### DISCUSSION TITLE
[javascript] support ecmascript decorator?

### DISCUSSION THREAD

**Discussion by wjljack** [pmd/pmd#3017] [Q&A]
support ecmascript decorator?

**Comment by oowekyala** [comment]
Please edit your question to clarify what you want.

**Comment by adangel** [comment]
@wjljack Are you referring to https://github.com/tc39/proposal-decorators ?
PMD uses https://github.com/mozilla/rhino to parse javascript/ecmascript and decorators are probably not supported by rhino yet. As far as I understand, decorators are a draft proposal and not standardized yet.
